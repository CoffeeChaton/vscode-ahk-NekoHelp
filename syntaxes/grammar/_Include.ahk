;https://www.autohotkey.com/docs/v1/lib/_Include.htm#Examples
#Include C:\My Documents\Scripts\Utility Subroutines.ahk ;Comment
#Include %A_ScriptDir% ;Comment
#Include C:\My Scripts ;Comment

#Include FileOrDirName
#Include <LibName>
#IncludeAgain FileOrDirName

#Include *i FileOrDirName
#Include *i <LibName>
#IncludeAgain *i FileOrDirName

#IncludeAgain %A_ScriptDir%
#IncludeAgain %A_AppData%
#IncludeAgain %A_AppDataCommon%
#IncludeAgain %A_LineFile%
