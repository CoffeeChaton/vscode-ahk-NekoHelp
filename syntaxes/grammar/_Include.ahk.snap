>;https://www.autohotkey.com/docs/v1/lib/_Include.htm#Examples
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>#Include C:\My Documents\Scripts\Utility Subroutines.ahk ;Comment
#^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#        ^ source.ahk string.entity.name.namespace.import.ahk
#         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
#                                                        ^ source.ahk string.entity.name.namespace.import.ahk meta.comment.line.semicolon.ahk
#                                                         ^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>#Include %A_ScriptDir% ;Comment
#^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#        ^ source.ahk string.entity.name.namespace.import.ahk
#         ^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
#                      ^ source.ahk string.entity.name.namespace.import.ahk meta.comment.line.semicolon.ahk
#                       ^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>#Include C:\My Scripts ;Comment
#^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#        ^ source.ahk string.entity.name.namespace.import.ahk
#         ^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
#                      ^ source.ahk string.entity.name.namespace.import.ahk meta.comment.line.semicolon.ahk
#                       ^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>
>#Include FileOrDirName
#^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#        ^ source.ahk string.entity.name.namespace.import.ahk
#         ^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>#Include <LibName>
#^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#        ^ source.ahk string.entity.name.namespace.import.ahk
#         ^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>#IncludeAgain FileOrDirName
#^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#             ^ source.ahk string.entity.name.namespace.import.ahk
#              ^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>
>#Include *i FileOrDirName
#^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#        ^ source.ahk string.entity.name.namespace.import.ahk
#         ^^ source.ahk string.entity.name.namespace.import.ahk keyword.operator.arithmetic.ahk
#           ^^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>#Include *i <LibName>
#^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#        ^ source.ahk string.entity.name.namespace.import.ahk
#         ^^ source.ahk string.entity.name.namespace.import.ahk keyword.operator.arithmetic.ahk
#           ^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>#IncludeAgain *i FileOrDirName
#^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#             ^ source.ahk string.entity.name.namespace.import.ahk
#              ^^ source.ahk string.entity.name.namespace.import.ahk keyword.operator.arithmetic.ahk
#                ^^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>
>#IncludeAgain %A_ScriptDir%
#^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#             ^ source.ahk string.entity.name.namespace.import.ahk
#              ^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>#IncludeAgain %A_AppData%
#^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#             ^ source.ahk string.entity.name.namespace.import.ahk
#              ^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>#IncludeAgain %A_AppDataCommon%
#^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#             ^ source.ahk string.entity.name.namespace.import.ahk
#              ^^^^^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>#IncludeAgain %A_LineFile%
#^^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk keyword.control.import.ahk
#             ^ source.ahk string.entity.name.namespace.import.ahk
#              ^^^^^^^^^^^^ source.ahk string.entity.name.namespace.import.ahk
>